project-koku/koku
# [COST-3901] 'others' included in ordering by distributed cost & limit.

## Jira Ticket

[COST-3901](https://issues.redhat.com/browse/COST-3901)

## Description

This change will always put others at the end. 

## Testing

1. Load test customer data
2. On main go to:
http://localhost:8000/api/cost-management/v1/reports/openshift/costs/?group_by[project]=*&filter[limit]=2&order_by[distributed_cost]=asc

Notice how others is not at the end.

3. Switch to this branch
4. `make clear-cache`
5. Refresh page and notice others goes to the end. 

## Notes

...
